backend/app/testbot/data_loader.py

```
@@ -30,38 +30,37 @@ def load_file(file_path: str) -> List[str]:
-        # Add the last chunk if it exists
+
+        # Append the remaining chunk if any
         if current_chunk:
             chunks.append('\n'.join(current_chunk))
-        
+
         return chunks
+
     except Exception as e:
-        logger.error(f"Error loading markdown: {e}")
+        logger.error("Error loading markdown: %s", e)
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)